WPMedia/fusion-news-theme-blocks
# THEMES-760: Identity Blocks | Login links block

## Description

This PR adds in the Login links block that accompanies the sign up and log in forms. It also corrects some styling issues with the single column layout related to the display of these blocks.

## Jira Ticket

- [THEMES-760](https://arcpublishing.atlassian.net/browse/THEMES-760)

## Acceptance Criteria

Login links block is converted to v2.

## Test Steps

1. Checkout this branch `git checkout THEMES-760`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/identity-block,@wpmedia/single-column-layout-block`
3. Create a page using the single column layout.
4. Add the Identity Log In - Arc Block and the Identity Login Links - Arc Block to the "body" section of the page.
5. In the Identity Login Links - Arc Block make sure "Show Forgot Password link" and "Show Sign up link" are checked. You can customized the URLs for each as well.
6. Make sure the login links appear below the form as seen on the designs (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=12874-178005&mode=design&t=JptOC1IZ0G9wCmAD-4).
7. Now replace Identity Log In - Arc Block with Identity Sign Up - Arc Block.
8. In the Identity Login Links - Arc Block make sure "Show Login link" is checked. You can customized the URL as well.
9. Make sure the login link appear below the form as seen on the designs (https://www.figma.com/file/vWVGgZfZVBdSX7ACo3kmjr/Enhanced-Styling-Design-System?type=design&node-id=8411-119293&mode=design&t=BeXCdtxm1ekWDLRJ-4).

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-760]: https://arcpublishing.atlassian.net/browse/THEMES-760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
